src/bodies/satellites.jl

```diff
@@ -0,0 +1,268 @@
+abstract type NaturalSatellite <: CelestialBody end
+
+export deimos, Deimos, phobos, Phobos, luna, Luna, moon, charon, Charon
+
+# Mars
+
+struct Phobos <: NaturalSatellite end
+const phobos = Phobos()
+Base.show(io::IO, ::Phobos) = print(io, "Phobos")
+parent(::Phobos) = mars_barycenter
+naifid(::Phobos) = 401
+
+struct Deimos <: NaturalSatellite end
+const deimos = Deimos()
+Base.show(io::IO, ::Deimos) = print(io, "Deimos")
+parent(::Deimos) = mars_barycenter
+naifid(::Deimos) = 402
+
+# Earth
+
+struct Luna <: NaturalSatellite end
+const luna = Luna()
+const moon = luna
+Base.show(io::IO, ::Luna) = print(io, "Luna")
+parent(::Luna) = earth_barycenter
+naifid(::Luna) = 301
+
+# Jupiter
+
+const JUPITER_SATELLITE_NAMES = (
+    "Io",
+    "Europa",
+    "Ganymede",
+    "Callisto",
+    "Amalthea",
+    "Himalia",
+    "Elara",
+    "Pasiphae",
+    "Sinope",
+    "Lysithea",
+    "Carme",
+    "Ananke",
+    "Leda",
+    "Thebe",
+    "Adrastea",
+    "Metis",
+    "Callirrhoe",
+    "Themisto",
+    "Magaclite",
+    "Taygete",
+    "Chaldene",
+    "Harpalke",
+    "Kalyke",
+    "Iocaste",
+    "Erinome",
+    "Isonoe",
+    "Praxidike",
+    "Autonoe",
+    "Thyone",
+    "Hermippe",
+    "Aitne",
+    "Eurydome",
+    "Euanthe",
+    "Euporie",
+    "Orthosie",
+    "Sponde",
+    "Kale",
+    "Pasithee",
+    "Hegemone",
+    "Mneme",
+    "Aoede",
+    "Thelxinoe",
+    "Arche",
+    "Kallichore",
+    "Helike",
+    "Carpo",
+    "Eukelade",
+    "Cyllene",
+    "Kore",
+    "Herse",
+    "Dia",
+)
+
+for (i, body) in enumerate(JUPITER_SATELLITE_NAMES)
+    typ = Symbol(body)
+    sym = Symbol(lowercase(body))
+    id = 500 + i
```

Comment:
I've concerns about how future-proof this `id` can be.  They've just discovered [a bunch of new Jupiter satellites](https://www.cnet.com/news/jupiter-has-dozen-new-moons-including-one-oddball-valetudo/).

Reply:
They are assigned by [NAIF](https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/FORTRAN/req/naif_ids.html) and my understanding is that they do not change once assigned.

Reply:
Oh, I didn't know, then I think it's good to follow this standard.